src/torchcodec/_core/ops.py

```diff
@@ -47,14 +50,19 @@ def load_torchcodec_shared_libraries():
         custom_ops_library_name = f"libtorchcodec_custom_ops{ffmpeg_major_version}"
         pybind_ops_library_name = f"libtorchcodec_pybind_ops{ffmpeg_major_version}"
         try:
-            torch.ops.load_library(_get_extension_path(decoder_library_name))
+            decoder_library_path = _get_extension_path(decoder_library_name)
+            torch.ops.load_library(decoder_library_path)
             torch.ops.load_library(_get_extension_path(custom_ops_library_name))
 
             pybind_ops_library_path = _get_extension_path(pybind_ops_library_name)
             global _pybind_ops
             _pybind_ops = _load_pybind11_module(
                 pybind_ops_module_name, pybind_ops_library_path
             )
+            global variant
+            global core_library_path
+            variant = ffmpeg_major_version
+            core_library_path = decoder_library_path
```

Comment:
Let's return them instead of having them as `global`.

Also, `ffmpeg_major_version` is a good name, so we can keep using it for the variable that we expose as `torchcodec.ffmpeg_major_version`

Reply:
done